tests/test_utils.py

```diff
@@ -112,6 +114,32 @@ def test_get_full_version_unexpected(monkeypatch):
         assert get_full_version() == __version__ + ".x"
 
 
+def test_rename_file():
+    with tempfile.TemporaryDirectory() as tmpdir_name:
+        filepath = os.path.join(tmpdir_name, "test_file.name")
+        base, ext = os.path.splitext(filepath)
+
+        # Create file on disk.
+        with open(filepath, "w") as f:
+            f.write("Test file")
+
+        # Case 1: move=False should return the new file name with apended timestamp.
+        renamed_file = rename_file(filepath, move=False)
+        timestamp = datetime.now().strftime("%y%m%d_%H%M")
+        assert renamed_file.startswith(f"{base}_{timestamp}") and renamed_file.endswith(
```

Comment:
In another suggestion I mention how you can form the exact name.

Reply:
Done.